lib/src/cognito_user.dart

```diff
@@ -657,6 +657,18 @@ class CognitoUser {
     return data;
   }
 
+  Future<void> completeNewPasswordChallenge(String newPassword) async {
```

Comment:
Any reason we don't want to pass back some status of whether the new password challenge was successful or failed?

Reply:
I don't have any issue passing back the response personally. However, I understood this lib to use exceptions for flow control. So my thoughts were that if there was a problem that the client would except and that you'd have to appropriately deal with that.

Still, I don't have any reservations about responding with a good or bad signal to the caller. @jonsaw what are your thoughts on this?